decidim-generators/lib/decidim/generators/app_templates/database.yml.erb

```diff
@@ -82,6 +82,7 @@ test:
 #
 #   production:
 #     url: <%%= ENV['DATABASE_URL'] %>
-#
+
 production:
-  url: <%%= ENV['DATABASE_URL'] %>
+  <<: *default
+database: <%= app_name %>_production
```

Comment:
Shouldn't this line be indented? So that it's inside the `production` key

Reply:
@mrcasals Indeed, let it be indented.